website/content/docs/platform/k8s/helm/examples/ha-with-consul.mdx

```diff
@@ -20,7 +20,7 @@ server:
     GOOGLE_PROJECT: myproject
     GOOGLE_APPLICATION_CREDENTIALS: /vault/userconfig/my-gcp-iam/myproject-creds.json
 
-  extraVolumes:
+  Volumes:
     - type: secret
       name: my-gcp-iam
```

Comment:
I think this will need to be more like this:

```suggestion
  volumes:
    - name: userconfig-my-gcp-iam
      secret:
        defaultMode: 420
        secretName: my-gcp-iam

  volumeMounts:
    - mountPath: /vault/userconfig/my-gcp-iam
      name: userconfig-my-gcp-iam
      readOnly: true
```